src/main/java/school/lemon/changerequest/java/vehicles/Vehicle.java

```diff
@@ -0,0 +1,15 @@
+package school.lemon.changerequest.java.vehicles;
+
+abstract class Vehicle {
+
+    String make;
```

Comment:
make all fields private and implement all getters\setters for those fields here directly, there is no need for code duplication.